lib/iris/fileformats/grib/__init__.py

```diff
@@ -55,6 +55,7 @@
 
 #: Set this flag to True to enable support of negative forecast periods
 #: when loading and saving GRIB files.
+#: .. deprecated:: 1.10
```

Comment:
An extra "#:" spacing line is wanted before the 'deprecated' line, to make this display nicely.